wal-css/src/parser/parsing_functions.rs

```diff
@@ -0,0 +1,387 @@
+use nom::branch::alt;
+use nom::bytes::complete::{tag, take_till, take_till1, take_until};
+use nom::character::complete::{multispace0, multispace1};
+use nom::error::{Error, ErrorKind, ParseError};
+use nom::multi::{separated_list0, separated_list1};
+use nom::sequence::{delimited, separated_pair, tuple};
+use nom::{combinator::map, sequence::pair, Err, IResult};
+
+use super::types::*;
+
+/// parses css string into Stylesheet object
+pub fn parse_stylesheet(i: &str) -> IResult<&str, Stylesheet> {
+    map(separated_list0(multispace1, p_section), |sections| {
```

Comment:
wydaje mi się że w ruscie |sections| { Stylesheet::new(sections) } to to samo co Stylesheet::new. Jak jest lepiej nwm